ldms/src/sampler/netlink/netlink-notifier.c

```diff
@@ -4510,6 +4511,94 @@ static void *dump_pids(void *vp)
 	return NULL;
 }
 
+char *dummy_pidenv[] = {"a=1","b=1", NULL};
+proc_info_t test_pi = {
+	.next = NULL,
+	.cmdline = "/bin/sleep 10",
+	.pid = 123,
+	.uid = 1023,
+	.gid = 1024,
+	.euid = 1025,
+	.start.tv_sec = 100000000,
+	.start.tv_usec = 1,
+	.start_tick = 1234567,
+	.kernel_thread = false,	/* true if a kernel thread */
+	.exe = "/bin/sleep",
+	.serno = 601,
+	.rm_type = RM_NONE,	/* replace with lsf or slurm as neeced */
+	.pidenv = dummy_pidenv,
+	.jobid = "123_45",
+	.is_thread = false,	/* replace with true as needed */
+	.excluded = false,
+	.excluded_short = false,
+	.emitted = EMIT_NONE,
+	.duration = 21345
+};
+
+void dump_schemas(forkstat_t *ft)
+{
+	if (!ft || !ft->schema_dump) {
+		return;
+	}
+	proc_info_t *info = &test_pi;
+	jbuf_t jbd = jbuf_new();
+	jbuf_t jb = jbd;
+	FILE *out = fopen(ft->schema_dump, "w");
+	if (!out) {
+		PRINTF("-J error: cannot open file %s\n", ft->schema_dump);
+		return;
+	}
+	PRINTF("dumping schema information to %s. configured format is %lu\n",
+		ft->schema_dump, ft->format);
+	int oldfmt = ft->format;
+	char *end = NULL;
+	unsigned long max_format = strtoul(default_format, &end, 10);
+	unsigned long k;
+	for (k = 0; k <= max_format; k++) {
+		ft->format = k;
+		fprintf(out, "\noptions: format=%lu %s %s\n",
+				ft->format, ft->prod_field, ft->compid_field);
+
+		//	case RM_NONE:
```

Comment:
@baallan, this is a nit, but you probably want to remove the tab from the comment.